p2p/libp2p/netMessenger.go

```diff
@@ -686,6 +687,46 @@ func (netMes *networkMessenger) Bootstrap() error {
 	return err
 }
 
+// WaitForConnections will wait the maxWaitingTime duration or until the target connected peers was achieved
+func (netMes *networkMessenger) WaitForConnections(maxWaitingTime time.Duration, minNumOfPeers uint32) {
+	startTime := time.Now()
+	defer func() {
+		log.Debug("networkMessenger.WaitForConnections",
+			"waited", time.Since(startTime), "num connected peers", len(netMes.ConnectedPeers()))
+	}()
+
+	if minNumOfPeers <= 0 {
```

Comment:
Since `minNumOfPeers` is a `uint32` variable, checking if it is `<= 0` is redundant, since it cannot be negative. 
Perhaps change it to  `minNumOfPeers == 0` ?

Reply:
right, remained from the initial implementation in which I've used an int